CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/CameraFragment.kt

```diff
@@ -132,7 +129,6 @@ class CameraFragment : Fragment() {
         override fun onDisplayChanged(displayId: Int) = view?.let { view ->
             if (displayId == this@CameraFragment.displayId) {
                 Log.d(TAG, "Rotation changed: ${view.display.rotation}")
-                preview?.setTargetRotation(view.display.rotation)
```

Comment:
Because we are removing `AutoFitTextureView`, which handled the rotations, the preview is now broken after the device is rotated. It's not an easy fix, and eventually it should be solved by `PreviewView`, so I suggest removing the `android:configChanges` tag from the activity in the Manifest for now.

Reply:
The `PreviewView` seems to lack an instance method `.setTargetRotation()` - only the builder has it.

Reply:
Have removed `android:configChanges`. When clicking the tiny rotate-icon on the system UI, it changes the layout orientation - just not automatically (which might be the intended behavior).